velox/vector/arrow/Bridge.cpp

```diff
@@ -511,6 +525,59 @@ VectorPtr createFlatVector(
 using WrapInBufferViewFunc =
     std::function<BufferPtr(const void* buffer, size_t length)>;
 
+VectorPtr createStringFlatVectorFromUtf8View(
+    memory::MemoryPool* pool,
+    const TypePtr& type,
+    BufferPtr nulls,
+    const ArrowArray& arrowArray,
+    WrapInBufferViewFunc wrapInBufferView) {
+  int num_buffers = arrowArray.n_buffers;
```

Comment:
`int64_t` for consistency with arrow abi

Reply:
Done